test/feeds.test.js

```diff
@@ -0,0 +1,67 @@
+const request = require('supertest');
+
+const app = require('../src/backend/web/app');
+const Feed = require('../src/backend/data/feed');
+const { addFeed } = require('../src/backend/utils/storage');
+const hash = require('../src/backend/data/hash');
+
+describe('test /feeds endpoint', () => {
+  const createdItems = 150;
+
+  // Array of feeds
+  const feeds = [...Array(createdItems).keys()].map(item => {
+    const url = `http://telescope${item}.cdot.systems`;
+    return {
+      id: hash(url),
+      author: 'foo',
+      url,
+    };
+  });
+
+  beforeAll(() => Promise.all(feeds.map(feed => addFeed(feed))));
+
+  it('requests feeds', async () => {
+    const res = await request(app).get('/feeds');
+
+    expect(res.status).toEqual(200);
+    expect(res.get('Content-type')).toContain('application/json');
+    expect(res.body.length).toBe(createdItems);
+    expect(res.body instanceof Array).toBe(true);
+  });
+});
+
+describe('test /posts/:guid responses', () => {
+  const existingGuid = 'http://existing-guid';
+  const missingGuid = 'http://missing-guid';
+
+  // an object to be added for testing purposes
+  const addedFeed = new Feed('foo', existingGuid);
+
+  // an object, expected to be returned by a correct query
+  const receivedFeed = {
+    author: 'foo',
+    url: 'http://existing-guid',
```

Comment:
Here and in the line below, use your `existingUrl` variable from above vs. a string literal (less chance of getting it wrong that way).